mozldap.go

```diff
@@ -31,26 +31,17 @@ func (cli *Client) Close() {
 // default are used (tls1.2, secure verify, ...).
 //
 // * uri is a connection string to the ldap server, eg. `ldaps://example.net:636/dc=example,dc=net`
-//
 // * username is a bind user, eg. `uid=bind-bob,ou=logins,dc=mozilla`
-//
 // * password is a password for the bind user
-//
 // * cacertpath is the path to a file containing trusted root certificates
-//
 // * tlsconf is a Go TLS Configuration
-//
 // * starttls requires that the LDAP connection is opened insecurely but immediately switched to TLS using the StartTLS protocol.
```

Comment:
Those extra newlines were put here for formatting purpose on GoDoc. I can't remember if they are necessary or not.